packages/kbn-discover-utils/src/components/custom_control_columns/degraded_docs_control.tsx

```diff
@@ -119,3 +129,48 @@ const DegradedDocs = ({
     />
   );
 };
+
+const EnableESQLDegradedDocsControl = ({
+  addIgnoredMetadataToQuery,
+  Control,
+}: {
+  addIgnoredMetadataToQuery: DegradedDocsControlProps['addIgnoredMetadataToQuery'];
+  Control: RowControlComponent;
+}) => {
+  const [isPopoverOpen, { off: closePopover, toggle: togglePopover }] = useBoolean(false);
+
+  return (
+    <EuiPopover
+      anchorPosition="upCenter"
+      button={
+        <Control
+          css={{ color: euiThemeVars.euiColorDisabledText }}
+          data-test-subj="docTableDegradedDocDisabled"
+          iconType="indexClose"
+          label={actionsHeaderAriaLabelDegradedAction}
+          onClick={togglePopover}
```

Comment:
I think a tooltip would be still helpful to have. Otherwise it's not clear that this grayed out icon is clickable.

Reply:
Do you mean something like "Click to enable degraded docs for this search?". Maybe if we do so we can even remove the need for a popover and have a single click action to enable degrade docs, wdyt?

Reply:
I like how it would just be simpler for the user and the design is more consistent, shall we go for it?

https://github.com/user-attachments/assets/2b0ac39b-eab8-456c-8246-84574a10e548

Reply:
Looks good to me

Reply:
Agree, I updated with [refactor(discover): add tooltip and direct action to degrade docs con…](https://github.com/elastic/kibana/pull/195630/commits/961dbde83f29e98ba13d4e4c3996542161db95bf)